tests/trainer/test_dataloaders.py

```diff
@@ -149,7 +146,6 @@ class CurrentTestModel(LightTrainDataloader, TestModelBase):
 
 def test_train_val_dataloaders_passed_to_fit(tmpdir):
     """ Verify that train & val dataloader can be passed to fit """
-    tutils.reset_seed()
 
```

Comment:
we need seeds on top of every test... why are you removing them?

Reply:
Here the seed is is the function run...